docs/data/data-grid/virtualization/virtualization.md

```diff
@@ -38,7 +38,7 @@ You may want to turn it off to be able to test the data grid with a headless bro
 <DataGrid {...data} disableVirtualization />
 ```
 
-:::info
+:::success
```

Comment:
This callout sounds more like a warning to me since it points out the drawbacks of disabling virtualization

Reply:
cc @samuelsycamore

Reply:
Applied